### ISSUE TITLE
Error: Video support is permanently disabled in this executable.

### ISSUE THREAD

**Issue by mralexgray** [transitive-bullshit/ffmpeg-generate-video-preview#4]
On a Mac with ffmpeg and gifski installed via homebrew..  you get this error when trying to make a gif....

`Error: Video support is permanently disabled in this executable.`

from [gifski's site](https://github.com/ImageOptim/gifski), it sounds like a hard issue to fix..

**Comment by carlos-mg89** [comment]
Has anyone been able to make this library work with gifski? It's working like a charm for me, with the PNG preview. But I do really would like the GIF one. 

I'm getting this very same error on Ubuntu, after having installed gifski.

Basically, these were the commands I ran:

```
npm install -g ffmpeg-generate-video-preview
sudo apt install ffmpeg
sudo snap install gifski
```

And this was the output I've got when trying the GIF preview:

```
generate-video-preview S01E01.avi output.gif --width 320 --num-frames 30 --gif-fps 4
{ cmd: 'ffmpeg -i S01E01.avi -y -vsync vfr -q:v 2 -vf select=not(mod(n\\,962)),scale=320:-1 /tmp/44f3542f931432667f22082def598757/frame-%d.png' }
gifski -o output.gif --fps 4 --quality 80 --quiet /tmp/44f3542f931432667f22082def598757/frame-*.png
generate-preview error { Error: Command failed: /bin/sh -c gifski -o output.gif --fps 4 --quality 80 --quiet /tmp/44f3542f931432667f22082def598757/frame-*.png
error: Unable to find the input file: "/tmp/44f3542f931432667f22082def598757/frame-1.png"


    at makeError (/home/carlos/.nvm/versions/node/v8.15.0/lib/node_modules/ffmpeg-generate-video-preview/node_modules/execa/index.js:172:9)
    at Promise.all.then.arr (/home/carlos/.nvm/versions/node/v8.15.0/lib/node_modules/ffmpeg-generate-video-preview/node_modules/execa/index.js:277:16)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
  code: 1,
  stdout: '',
  stderr: 'error: Unable to find the input file: "/tmp/44f3542f931432667f22082def598757/frame-1.png"\n',
  failed: true,
  signal: null,
  cmd: '/bin/sh -c gifski -o output.gif --fps 4 --quality 80 --quiet /tmp/44f3542f931432667f22082def598757/frame-*.png',
  timedOut: false,
  killed: false }
(node:43383) UnhandledPromiseRejectionWarning: Error: Command failed: /bin/sh -c gifski -o output.gif --fps 4 --quality 80 --quiet /tmp/44f3542f931432667f22082def598757/frame-*.png
error: Unable to find the input file: "/tmp/44f3542f931432667f22082def598757/frame-1.png"


    at makeError (/home/carlos/.nvm/versions/node/v8.15.0/lib/node_modules/ffmpeg-generate-video-preview/node_modules/execa/index.js:172:9)
    at Promise.all.then.arr (/home/carlos/.nvm/versions/node/v8.15.0/lib/node_modules/ffmpeg-generate-video-preview/node_modules/execa/index.js:277:16)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:43383) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:43383) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

There seems to be an issue with reading the /tmp/44f3542f931432667f22082def598757 folder, where the PNGs are created (they are there, that's verified). The frame "/tmp/44f3542f931432667f22082def598757/frame-1.png" is there as well.

Thank you in advance! And thanks anyway to the project's developer. It's really close to work nicely.

**Comment by carlos-mg89** [comment]
I've found the solution. In the official website of gifski, there's the download link: https://gif.ski/

If you install gifski with sudo snap install gifski, you'll get an old version. I got the 1.2.7. However, if you download gifski from their website, you get the version 1.4.0.

They have Windows, Mac and Linux. I've tested it only in Ubuntu 20.10.

Install the .deb file (if on Ubuntu / Debian based distros), and then, generate-video-preview will work perfectly fine.
